### PR TITLE
Add Aeson instances

### PR DESCRIPTION
**Pull request by peter-mlabs**

### CODE REVIEW

**Review by kozross — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,12 @@
 
 This format is based on [Keep A Changelog](https://keepachangelog.com/en/1.0.0).
 
+## 3.6.1 -- 2022-09-20
+
+### Added
+
+* `Aeson` instances for Plutus types (in the `Plutarch.Orphans` module)
```
Comment (kozross) [inline CHANGELOG.md:9]: Note the `newtype` being exported from that module, as this is fairly unusual. Normally, `Orphans` modules don't export anything.

File: src/Plutarch/Orphans.hs [hunk 2]
```diff
@@ -62,3 +102,165 @@ instance PTryFrom PData (PAsData PScriptHash) where
...
+        (Aeson.FromJSON MintingPolicy)
+
+deriving via
+    (AsBase16Codec Script)
+    instance
+        (Aeson.ToJSON Script)
+deriving via
+    (AsBase16Codec Script)
+    instance
+        (Aeson.FromJSON Script)
```
Comment (kozross) [inline src/Plutarch/Orphans.hs:266]: All require `since`s.

**Review by kozross — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (peter-mlabs) [reply CHANGELOG.md:9]: I don't think we actually need that export...

Comment (peter-mlabs) [reply CHANGELOG.md:9]: I'll check

File: src/Plutarch/Orphans.hs (hunk 2 above)
Comment (peter-mlabs) [reply src/Plutarch/Orphans.hs:266]: ooft. I noticed this, went to add to the change log, and then immediately forgot :sweat_smile:

File: CHANGELOG.md (hunk 1 above)
Comment (peter-mlabs) [reply CHANGELOG.md:9]: Don't need it. Its been removed.

**Review by kozross — APPROVED** [review]

**Pull request merged**